pageserver/Cargo.toml

```diff
@@ -61,6 +61,7 @@ serde_with.workspace = true
 sysinfo.workspace = true
 tokio-tar.workspace = true
 thiserror.workspace = true
+time.workspace = true
```

Comment:
We already pull in chrono, which also does daterime formatting. Consider using it instead.